.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -13,56 +14,169 @@ env:
-      - select: "auditbeat - run_auditbeat_win_tests"
-        key: "run_auditbeat_win_tests"
-        options:
-          - label: "True"
-            value: "true"
-          - label: "False"
-            value: "false"
-        default: "false"
-    if: "build.source == 'ui'"
-
-  - wait: ~
-    if: "build.source == 'ui'"
-    allow_dependency_failure: false
-
-  - label: ":linux: Load dynamic auditbeat pipeline"
-    key: "auditbeat-pipeline"
-    command: ".buildkite/auditbeat/generate_auditbeat_pipeline.sh"
-    notify:
-      - github_commit_status:
-          context: "${BEATS_PROJECT_NAME}: Load dynamic pipeline's steps"
+  - group: "Auditbeat Mandatory Testing"
+    key: "auditbeat-mandatory-tests"
+
+    steps:
+      - label: ":ubuntu: Auditbeat Unit Tests"
+        command: "cd ${BEATS_PROJECT_NAME} && mage unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":rhel: Auditbeat Unit Tests"
+        command:
+          - "cd ${BEATS_PROJECT_NAME} && mage unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_RHEL9}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":windows: Auditbeat Win-2016 Auditbeat Unit Tests"
+        command: "mage -d ${BEATS_PROJECT_NAME} unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2016}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 200
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":windows: Auditbeat Win-2022 Unit Tests"
+        command: "mage -d ${BEATS_PROJECT_NAME} unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2022}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 200
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":linux: Auditbeat Crosscompile"
+        command: "make -C auditbeat crosscompile"
+        env:
+          GOX_FLAGS: "-arch amd64"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+
+  - group: "Auditbeat ARM Tests"
+    key: "auditbeat-extended-tests-arm"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+    steps:
+      - label: ":arm: ARM64 Unit Tests"
+        key: "auditbeat-extended-arm64-unit-tests"
+        command: "cd ${BEATS_PROJECT_NAME} && mage unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_ARM64}"
```

Comment:
This env var is missing from the `env:` section, resulting in an empty string and falling back to the default image, see https://buildkite.com/elastic/beats/builds/4716#018ea906-ee6d-4a0c-8b8c-fe1600199bda

Reply:
Good catch! fixed.